source/userguide/kb.rst

```diff
@@ -813,3 +813,122 @@ We can perform the same search using the :meth:`query()` API as well.
+
+.. code:: python
+
+  QUESTION_ANSWERER_CONFIG = {
+      "model_type": "embedder",
+      "model_settings": {
+          "embedder_type": "bert",
+          "embedding_fields": {
+              "faq_data": ["question", "answer"]
+          }
+      }
+  }
+
+There are three available model types which leverage semantic embedders. The specified ``model_type`` is also the default ``query_type`` for all question answering calls for your application, but you can always pass in a different query type to your ``qa.get()`` command as desired.
+
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+  | Model type            | Description                                                                                                |
+  +=======================+============================================================================================================+
+  | embedder              | For leveraging deep semantic embedders. This option allows for using deep semantic embedders like          |
+  |                       | sentence bert for doing vector-based retrieval                                                             |
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+  | embedder_keyword      | For leveraging deep semantic embedders and text signals for structured search on keywords or short spans   |
+  |                       | of text. Glove as an additional signal has shown improvements in this use case.                            |
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+  | embedder_text         | For leveraging deep semantic embedders and text signals for unstructured search on larger paragraphs or    |
+  |                       | passages. BERT as an additional signal has shown improvements in this use case.                            |
+  +-----------------------+------------------------------------------------------------------------------------------------------------+
+
+
+The two included embedder types are `Sentence BERT <https://github.com/UKPLab/sentence-transformers>`_ and `GloVe <https://nlp.stanford.edu/projects/glove/>`_. 
+
+Sentence BERT is a modification of the pretrained BERT network that use siamese and triplet network structures to derive semantically meaningful sentence embeddings that can be compared using cosine-similarity. To use this embedder type, you can specify ``bert`` as the ``embedder_type`` in the ``model_settings``.
```

Comment:
```suggestion
Sentence-BERT is a modification of the pretrained BERT network that use siamese and triplet network structures to derive semantically meaningful sentence embeddings that can be compared using cosine-similarity. To use this embedder type, you can specify ``bert`` as the ``embedder_type`` in the ``model_settings``.
```

Reply:
@arushir - Is there a chance that we might ever support the vanilla BERT as a built-in option? If there's a chance for that, I'd prefer calling this ``embedder_type`` as 'sentence-bert' (instead of just 'bert'). But if we're fairly certain that this adaption of BERT is the only one we'll natively support for question answering, I'm OK with calling it 'bert' for simplicity.